### PR TITLE
Icon updates

### PR DESCRIPTION
**Pull request by rossmoody**
Icon updates and refactoring are in this. It's mainly adding icons, revising a few that were formatting incorrectly and then tweaking the CSS that related to the icon in its environment. Also refactoring the naming convention to be more consistent. Been using: 
`[name]-[alt]-[shape]-[o]-[direction]` naming convention for icons. O for outline, omitting if it's a fill or glyph. Probably time for a larger refactor but should address in a followup.

Fixes https://github.com/brave/brave-browser/issues/2859
Provides the necessary icon to finish this PR correctly with the Pin outline icon: https://github.com/brave/brave-core/pull/1407

# Updated UI

## Old Wallet Panel 
image

## New Wallet Panel
screen shot 2019-01-24 at 12 16 00 am

## Before Verified Icon
image

## After Verified Icon
screen shot 2019-01-24 at 12 15 45 am

## Before Icons
screen shot 2019-01-24 at 12 16 44 am

## After Icons
image

### CODE REVIEW

**Comment by corymcdonald** [comment]
Are we going to cut a release and upload to NPM when this is completed?

**Comment by petemill** [comment]
Yes @corymcdonald this will result in a npm publish and major version change. @rossmoody has a PR in publishers that accommodates the changes, and that PR should be the one that pulls in the new version

**Comment by corymcdonald** [comment]
I was wondering if you could add a downloadIcon as well?

**Comment by rossmoody** [comment]
@corymcdonald I think that one is available currently: `DownloadIcon`

**Comment by cezaraugusto** [comment]
@rossmoody could you please rebase?

File: src/components/icons/pin.tsx [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
 /* tslint:disable */
 /* This Source Code Form is subject to the terms of the Mozilla Public
-* License. v. 2.0. If a copy of the MPL was not distributed with this file.
-* You can obtain one at http://mozilla.org/MPL/2.0/. */
+ * License. v. 2.0. If a copy of the MPL was not distributed with this file.
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import * as React from 'react'
-import GenerateComponentForGraphic from './template'
+import * as React from "react";
```
[the remark this answers is not in the document]
Comment (rossmoody) [reply src/components/icons/pin.tsx]: i really wish i could figure out why my vscode TSLint formatting is adding these occasionally. Will fix.

**Comment by NejcZdovc** [comment]
cherry-picked in 0.61 9f6474e1

**Pull request merged**